usr/share/rear/init/default/100_check_stale_nfs_mounts.sh

```diff
@@ -0,0 +1,20 @@
+# 100_check_stale_nfs_mounts.sh
+# Purpose is to have a simple test if we have stale NFS mount points which
+# could lead to a hangind ReaR session.
+# In case there is a stale NFS present we bail out with an error to get it fixed before
+# running ReaR again.
+# See issue https://github.com/rear/rear/issues/3109
+# To make the code more robust we take care of spaces and \ in the mount point naming.
+### example mount point /mnt/!@#$%^&**()_-=+{}|\
```

Comment:
Does this example have a space in the path? Bit unclear.

For me adding a note that spaces are shown as `\040` would have helped alleviating my worries about the `read` word splitting working properly.

Thanks for the more elaborate test!

Just for curiosity, does the `printf "%s\n"` do more than just using `<<<`?

Reply:
@schlomo  Added your favorite example ;-) and you're right `<<<` eliminates an additional `printf` command.

@rear/contributors If there are no more additional comments about the code then I would like to merge the script tomorrow afternoon.